ui/search-ui/search-geonames/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -19,4 +19,12 @@
 
     <service ref="geoCoder" interface="org.codice.ddf.ui.searchui.geocoder.GeoCoder"/>
 
+    <bean id="geoCoderLocal" class="org.codice.ddf.ui.searchui.geocoder.geonames.GeoNamesLocalIndex">
+        <property name="geoEntryQueryable" ref="geoEntryQueryable"/>
+    </bean>
+
+    <reference id="geoEntryQueryable" interface="org.codice.ddf.spatial.geocoding.GeoEntryQueryable"/>
+
+    <service ref="geoCoderLocal" interface="org.codice.ddf.ui.searchui.geocoder.GeoCoder"/>
```

Comment:
Without specifying rankings, I'm not sure which of the two services (online or offline) will be used by default.